compiler/modules/nback/module.bal

```diff
@@ -50,19 +53,37 @@ public function buildModule(bir:Module birMod, *Options options) returns [llvm:M
         stackGuard: llMod.addGlobal(llvm:pointerType("i8"), mangleRuntimeSymbol("stack_guard")),
         llInitTypes: createInitTypes(llContext)
     };  
+    bsexpr:Function[] funcs = [];
```

Comment:
As we discussed in the meeting, I think we should have an option to output the compiled module as BIR sexpr (instead of LLVM). Then nback should not depend on sexpr.